### ISSUE TITLE
Create a Validation category

### ISSUE THREAD

**Issue by sdelamo** [micronaut-projects/micronaut-docs-index#81]
Validation category should contain 

- Micronaut Validation
- Micronaut Hibernate Validator

**Comment by Richardson-e** [comment]
@sdelamo Is there a specific image I should use?

**Comment by sdelamo** [comment]
no, lets look for a svg without copyright which we can use to illustrate validation. maybe a checkmark.
